### PR TITLE
LVM data are now global

### PR DESCRIPTION
**Pull request by japokorn**
- created new singletons from lv_info and pv_info
- edited the code to use them

### CODE REVIEW

File: blivet/staticdata/lvm_info.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# volume_info.py
+# Backend code for populating a DeviceTree.
+#
+# Copyright (C) 2009-2015  Red Hat, Inc.
```
Comment (dwlehman) [inline blivet/staticdata/lvm_info.py]: Update this to just 2016.

**Comment by japokorn** [comment]
Updated Copyright date in volume_info.py from '2009-2015' to '2009-2016'

**Comment by dwlehman** [comment]
======================================================================
FAIL: test_run (populator_test.LVMFormatPopulatorTestCase)
Test lvm format populator.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/unittest/mock.py", line 1157, in patched
    return func(*args, **keywargs)
  File "/home/jenkins/workspace/blivet-master-PR/tests/populator_test.py", line 919, in test_run
    self.assertEqual(get_device_by_uuid.call_count, 3)
AssertionError: 2 != 3

File: blivet/staticdata/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .lvm_info import lvs_info
```
Comment (vpodzime) [inline blivet/staticdata/__init__.py]: why not also ``pvs_info`` here?

File: blivet/populator/helpers/lvm.py [hunk 3]
```diff
@@ -37,6 +37,9 @@
 from .devicepopulator import DevicePopulator
 from .formatpopulator import FormatPopulator
 
+from ...staticdata.lvm_info import lvs_info, pvs_info
+
+
```
Comment (vpodzime) [inline blivet/populator/helpers/lvm.py]: One blank line should be enough here.

Comment (japokorn) [inline blivet/populator/helpers/lvm.py]: Jenkins demanded two lines here.

Comment (vpodzime) [inline blivet/populator/helpers/lvm.py]: Are you sure? My bet is it was down below, not here. We have a lot of imports like this in many files and two blank lines are right below the ``log = logging.getLogger("blivet")`` line, not before ``import logging``.

**Comment by dwlehman** [comment]
(I thought I commented to this effect already, but I can't find it.)
Anyway, you also need to remove the old stuff from the lvm format populator test that modified `devicetree._pvs_cache` and `devicetree._lvs_cache` since those no longer exist.

**Comment by japokorn** [comment]
Pull request target branch has changed. Closing...
See https://github.com/rhinstaller/blivet/pull/349

**Pull request closed without merging**